src/layouts/Header.tsx

```diff
@@ -84,6 +87,8 @@ class Header extends React.Component<IProps, null> {
 
     const daoAvatarAddress = dao ? dao.address : null;
     const accountIsEnabled = getAccountIsEnabled();
+    const web3ProviderInfo = getWeb3ProviderInfo();
+    // const cachedWeb3ProviderInfo = this.props.getCachedWeb3ProviderInfo();
```

Comment:
so is this not actually used then?

Reply:
oh, I left that in because I anticipated that it is going to be used soon, but doubtfully in this PR.  So I can remove it.